regression/verilog/SVA/named_property1.desc

```diff
@@ -0,0 +1,9 @@
+CORE
+named_property1.sv
+--bound 0
+^\[main\.assert\.1\] always main\.x == 10: PROVED up to bound 0$
```

Comment:
I'd like that -- it would require splitting up the type checking phase such that the initial phase doesn't expand symbols marked as "macro". Will do eventually.

Reply:
#585